x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/details_step/details_step_form.tsx

```diff
@@ -195,6 +196,22 @@ export const DetailsStepForm: FC<CreateAnalyticsStepProps> = ({
           data-test-subj="mlAnalyticsCreateJobFlyoutDestinationIndexInput"
         />
       </EuiFormRow>
+      <EuiFormRow
+        label={i18n.translate('xpack.ml.dataframe.analytics.create.resultsFieldLabel', {
+          defaultMessage: 'Results field',
```

Comment:
We should include this in the summary step - if only when the user enters something other than the default of `ml`:

![image](https://user-images.githubusercontent.com/7405507/86808662-f6582d00-c072-11ea-96b2-02ac74026cfb.png)

Reply:
Good point - will show results field if it is set by the user. Updated in ba0dede3f530a09fd5219b8642bd2c1ca91ae907